contracts/0.8.9/utils/Versioned.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.9;
+
+
+import "../lib/UnstructuredStorage.sol";
+
+
+contract Versioned {
```

Comment:
Maybe introduce a drop-in replacement for `Autopetrifiable` here as well :eyes:

Reply:
It's already done in the constructor, line 28:

```solidity
CONTRACT_VERSION_POSITION.setStorageUint256(type(uint256).max);
```

This prevents calling `_initializeContractVersionTo1` which is expected to be called within an initializer. Maybe we can do it in a prettier way, though.